cmd/wego/add/cmd.go

```diff
@@ -56,6 +56,19 @@ func init() {
 func runCmd(cmd *cobra.Command, args []string) error {
 	params.Namespace, _ = cmd.Parent().Flags().GetString("namespace")
 
+	if params.Url != "" && len(args) > 0 {
+		return fmt.Errorf("you should choose either --url or the app directory")
+	}
+
+	if len(args) > 0 {
+		path, err := filepath.Abs(args[0])
+		if err != nil {
+			return fmt.Errorf("failed to get absolute path for the repo directory")
```

Comment:
missing append `err` ? it would help to describe what the actual problem is or at least what was the value of `args[0]` that made it fail. I forgot to add it when I committed this.

Reply:
thanks, missed that.